Index: Group_4_job_search_pro/Employer.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\r\n// Created by LENOVO on 21/02/2024.\r\n//\r\n\r\n#ifndef GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n#define GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n\r\n\r\nclass Employer {\r\n\r\n};\r\n\r\n\r\n#endif //GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Group_4_job_search_pro/Employer.h b/Group_4_job_search_pro/Employer.h
--- a/Group_4_job_search_pro/Employer.h	(revision f6781396074bf09c5e4fe31a4d42398f7c820615)
+++ b/Group_4_job_search_pro/Employer.h	(date 1709056749922)
@@ -6,7 +6,7 @@
 #define GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H
 
 
-class Employer {
+class Employer :{
 
 };
 
