Index: Group_4_job_search_pro/Employer.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\r\n// Created by LENOVO on 21/02/2024.\r\n//\r\n\r\n#ifndef GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n#define GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n\r\n\r\nclass Employer {\r\n\r\n};\r\n\r\n\r\n#endif //GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Group_4_job_search_pro/Employer.h b/Group_4_job_search_pro/Employer.h
--- a/Group_4_job_search_pro/Employer.h	(revision 1b152896d0f585fd629c361f9f9f1436a03af2b1)
+++ b/Group_4_job_search_pro/Employer.h	(date 1709056560631)
@@ -4,9 +4,21 @@
 
 #ifndef GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H
 #define GROUP_4_JOB_SEARCH_PRO_EMPLOYER_H
+using namespace std;
+#include "Job.h"
 
 
 class Employer {
+private:
+    Job* my_job;
+public:
+    void set(Job*item){my_job=item;}
+    Employer();
+    Employer(long id, string full_name,string mail,long phone_num,string paswoord)
+    void edit_job();
+    void publich_job();
+    void view_job();
+
 
 };
 
